js/packages/mobile-wallet-adapter-protocol/src/createMobileWalletProxy.ts

```diff
@@ -89,13 +89,6 @@ function handleMobileWalletRequest<TMethodName extends keyof MobileWallet>(
                     case 'mainnet-beta': { chain = 'solana:mainnet'; break; }
                 }
                 (params as Parameters<MobileWallet['authorize']>[0]).chain = chain;
-
-                if (signInPayload) {
```

Comment:
Checking understanding:

This is now removed because `params` will now just contain a `SignInPayload` type that doesn't need any conversion.